docs/docs/contextual-conversations.mdx

```diff
@@ -1,120 +1,182 @@
-other cases you will need to use stories and context to help the user achieve their goal.
-
-If we take the example of the “getting started” skill from Sara, we want to give them
-different information based on whether they've built an AI assistant before and are
-migrating from a different tool etc. This can be done quite simply with stories and
-the concept of [max history](./policies.mdx#max-history).
-
-```md
- ## new to rasa + built a bot before
- * how_to_get_started
-   - utter_getstarted
-   - utter_first_bot_with_rasa
- * affirm
-   - action_set_onboarding
-   - slot{"onboarding": true}
-   - utter_built_bot_before
- * affirm
-   - utter_ask_migration
- * deny
-   - utter_explain_rasa_components
-   - utter_rasa_components_details
-   - utter_ask_explain_nlucorex
- * affirm
-   - utter_explain_nlu
-   - utter_explain_core
-   - utter_explain_x
-   - utter_direct_to_step2
-
- ## not new to rasa + core
- * how_to_get_started
-   - utter_getstarted
-   - utter_first_bot_with_rasa
- * deny
-   - action_set_onboarding
-   - slot{"onboarding": false}
-   - utter_ask_which_product
- * how_to_get_started{"product": "core"}
-   - utter_explain_core
-   - utter_anything_else
+To create a context-aware conversational assistant, you will need to define
+how your assistant uses the conversation history to affect the next response.
+
+For example, if a user asks Sara, the Rasa bot, how to get started using Rasa,
+Sara gives the user
+different information based on whether they've built an AI assistant before or not:
+
+A conversation with a new user:
+```yaml
+User: How can I get started with Rasa?
+    Sara: To determine how I can help you best, I'm going to ask you a few questions.
+    Sara: Let's go. Are you new to Rasa?
+User: yes
+    Sara: And have you built a contextual assistant or a bot before?
 ```
 
-The above example mostly leverages intents to guide the flow, however you can also
-guide the flow with entities and slots. For example, if the user gives you the
-information that they're new to Rasa at the beginning, you may want to skip this
-question by storing this information in a slot.
-
-```md
-* how_to_get_started{"user_type": "new"}
-  - slot{"user_type":"new"}
-  - action_set_onboarding
-  - slot{"onboarding": true}
-  - utter_getstarted_new
-  - utter_built_bot_before
+A conversation with a returning user:
+```yaml
+User: How can I get started with Rasa?
+    Sara: To determine how I can help you best, I'm going to ask you a few questions.
+    Sara: Let's go. Are you new to Rasa?
+User: no
+    Sara: Ok, which product would you like to know more about? Rasa Open Source or Rasa X?
 ```
 
-For this to work, keep in mind that the slot has to be featurized in your `domain.yml`
-file. This time we can use the `text` slot type, as we only care about whether the
-[slot was set or not](./domain.mdx#slots/).
+This page is a guide on creating contextual conversation patterns using Rasa.
+
+## Contextual Conversation Patterns
```

Comment:
step-by-step guide again?

Reply:
although i'm actually not sure that even makes sense for this page. and if it's not step by step then i guess the page shouldn't be numbered either ?